data-format/message-body.md

```diff
@@ -16,9 +16,9 @@ See [Message Version](#message.md#version).
 
 ## Overview
 
-The message body is a component of the [message](#message.md).    
-The message body contains the encrypted data, called the [encrypted content](#encrypted-content.md).    
-The purpose of the message body is to define the structure containing the [encrypted content](#encrypted-content.md).
+The message body is a component of the [message](message.md).    
+The message body contains the encrypted data, called the [encrypted content](#encrypted-content).    
+The purpose of the message body is to define the structure containing the [encrypted content](#encrypted-content).
 
 The structure of the body depends on the content type:
 
```

Comment:
Juuust out of range of the comment space again, but the links for framed and non-framed data are swapped.